package.json

```diff
@@ -27,16 +27,23 @@
   },
   "license": "MIT",
   "devDependencies": {
+    "chai": "^3.5.0",
     "eslint": "^1.2.1",
     "eslint-config-standard": "^4.1.0",
     "eslint-plugin-standard": "^1.3.1",
     "grunt": "^0.4.1",
     "grunt-bump": "^0.5.0",
+    "grunt-cli": "^0.1.13",
```

Comment:
Added `grunt-cli` to avoid requiring global install of gulp. As good npm citizens we should not require such global installs by developers trying to run the tests :)